contracts/modules/TransferManager.sol

```diff
@@ -532,11 +530,10 @@ contract TransferManager is OnlyOwnerModule, BaseTransfer {
     * @param _wallet The target wallet.
     * @param _contract The address of the contract.
      */
-    function authoriseContractCall(address _wallet, address _contract) internal view {
+    function isNotManagedTokenOrLimitDisabled(address _wallet, address _contract) internal view {
```

Comment:
I'd maybe rename `isNotManagedTokenOrLimitDisabled` to `isNotKnownTokenOrLimitDisabled` since our `tokenPriceStorage` does not really "manage" tokens.

Reply:
To go one step further I think the name of this method should reflect that we are checking if the target contract is protected by the daily limit (not just known). So I would suggest something like `isNotCoveredByLimitOrLimitDisabled` returning a boolean and coupled with a require to make the code more readable.

Reply:
Maybe `isNotCoveredByLimit` also covers `..LimitDisabled` part of the name and we can keep it shorter to `notCoveredByDailyLimit`?

Reply:
Yes, with the `require(notCoveredByDailyLimit()...'  in the external methods like it :-)